docs/user/whats-new.asciidoc

```diff
@@ -1,27 +1,272 @@
+
+beta[] You can now create an index, and with a new set of drawing tools,
+add or delete documents. Use the drawing tools to create new data and add geographic
+context to your map, good for annotation or ad-hoc analysis.
+You can edit both new and existing layers (indices).
+Editing is not supported on layers with clustering, joins, or time filtering.
+
+[role="screenshot"]
+image::user/images/new-maps-edit.png[Map showing set of drawing tools to add or delete new data and add geographic context to your map]
+
+[float]
+==== Enhanced spatial filters
+
+Filtering is now more intuitive.  The spatial filters in *Maps* are enhanced to work
+with all layers on the map, meaning you no longer have to specify a target index pattern
+when configuring the filter. When using the distance filter, you will now see a
+distance value displayed as you draw the shape. This makes it easier to filter all
+documents that are within 10 kilometers of a specific location.
+
+[float]
+[[discover-7-14]]
+=== Discover&mdash;community-requested features
+
+*Discover* debuts several community-requested features: a multi-fields toggle,
+enhanced *Lens* integration, and source views in the document viewer.
+*Discover* also enhances the user experience with improvements to stability, performance,
+accessibility, and usability.  For more information on the most-used app in {kib}, refer to {kibana-ref}/discover.html[Discover].
+
+[float]
+[[canvas-7-14]]
+=== Canvas&mdash;improves SQL expression function
+
+The {es} SQL expression function in *Canvas* has changed to gracefully
+handle arrays in retrieved data. If the data for the SQL expression includes an array,
+the first entry of the array is returned in the result set.
+
+In addition, the SQL expression function now supports passing parameters into the query,
+making it easier to pass in variable values to your SQL queries on your workpad.
+
+[role="screenshot"]
+image::user/images/new-canvas.png[Canvas Elasticsearch SQL expression function]
+
+[float]
+[[ml-7-14]]
+
+=== Machine learning&mdash;maps,delayed data, & more
+
+[float]
+==== Choropleth maps for anomalies
+
+Choropleth maps are now available in the *Anomaly Explorer* and the *Data Visualizer*
+for fields such as country and region codes, zip codes, and states. The integration
+relies on a new service provided by the *Maps* plugin, which automatically
+identifies fields that store location data based on sample values or
+contextual knowledge about the data.
+For example, for jobs with region ISO code partitioning or influencer fields,
+a map showing the count of anomalies by location is displayed in the *Anomaly Explorer* .
+
+[role="screenshot"]
+image::user/images/new-ml-choropleth-map.png[Canvas Elasticsearch SQL expression function]
+
+For more details, see {ml-docs}/mapping-anomalies.html[Mapping anomalies by location].
+[float]
+==== Delayed data visualization for anomaly detection
+
+Delayed data are documents that are indexed after the datafeed
```

Comment:
There's additional docs for this one too. Can we add a "for more information" on this one too? Or else link it here:

```suggestion
{ml-docs}/ml-delayed-data-detection.html[Delayed data] are documents that are indexed after the {dfeed}
```